### PR TITLE
Add leaflet-fullscreen library definitions

### PR DESCRIPTION
**Pull request by wcomartin**

### CODE REVIEW

File: leaflet-fullscreen/leaflet-fullscreen.d.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Type definitions for Leaflet.fullscreen v1.3.0
+// Project: https://github.com/brunob/leaflet.fullscreen
```
Comment (horiuchi) [inline leaflet-fullscreen/leaflet-fullscreen.d.ts]: I guess Project is `https://github.com/Leaflet/Leaflet.fullscreen` rather than this.

Comment (wcomartin) [inline leaflet-fullscreen/leaflet-fullscreen.d.ts]: this is a fork with more options, its the version i use.

**Comment by horiuchi** [comment]
@DefinitelyTyped/contributors Uhmm.. We need the naming rule for this pattern of forked library.

**Comment by basarat** [comment]
@wcandillon the project should be on some module management system (prefer npm) with a unique name which will be the *true name* for your project. I know its a global module namespace war, but this project is to support global (named on npm) modules :rose:

**Comment by wcomartin** [comment]
Turns out the version i was using was "leaflet.fullscreen" not "leaflet-fullscreen" made the change.

**Comment by basarat** [comment]
Review note : Verified https://www.npmjs.com/package/leaflet.fullscreen / https://github.com/brunob/leaflet.fullscreen/blob/master/package.json 

Thanks @wcomartin :rose:

**Pull request merged**